src/pyird/utils/irdstream.py

```diff
@@ -201,7 +201,7 @@ def flatten(self, trace_path=None, extout='_fl', extin=None, mask=None):
         if not self.imcomb:
             extin_noexist, extout_noexist = self.check_existence(extin, extout)
             for i, fitsid in enumerate(tqdm.tqdm(extin_noexist)):
-                filen = self.rawdir/extin_noexist[i]
+                filen = self.anadir/extin_noexist[i]
```

Comment:
I noticed that `anadir` is better here.
(The additional modification for this PR is only this.)